src/main/java/BuffBuddy.java

```diff
@@ -0,0 +1,77 @@
+import com.google.gson.JsonObject;
+
+import command.Command;
+import core.History;
+import core.Storage;
+import core.Ui;
+import parser.Parser;
+import programme.ProgrammeList;
+
+public class BuffBuddy {
```

Comment:
This class seems to be handling multiple functions. You can consider splitting it into different files.